packages/datadog-plugin-openai/src/index.js

```diff
@@ -118,7 +118,7 @@ class OpenApiPlugin extends TracingPlugin {
     }
 
     // createChatCompletion, createCompletion
-    if (typeof payload.logit_bias === 'object' && payload.logit_bias) {
+    if (payload.logit_bias && typeof payload.logit_bias === 'object') {
```

Comment:
why not check for null directly ? like everywhere else ?

Reply:
Good catch. Fixed in 824d157

Reply:
Actually, I had left these truthy checks alone in this PR. But you comment made me revisit them, so I just pushed yet another PR addressing all of them in a similar manner: 4d5cfba